packages/button/src/css/index.ts

```diff
@@ -171,17 +171,21 @@ export default {
 
     '&:not([disabled]):hover': {
       border: `2px solid ${colorsBlue[5]}`,
-      background: transparentize(0.85, colorsPrimaryAction.textDarkTheme),
+      background: `rgba(${parseToRgb(
+        colorsPrimaryAction.textDarkTheme
+      )}, 0.85)`,
       color: colorsBlue[4]
     },
     '&:not([disabled]):focus': {
       border: `2px solid ${colorsBlue[5]}`,
-      background: transparentize(0.85, colorsPrimaryAction.textDarkTheme),
+      background: `rgba(${parseToRgb(
+        colorsPrimaryAction.textDarkTheme
+      )}, 0.85)`,
       color: colorsBlue[4]
     },
     '&:not([disabled]):active': {
       border: `2px solid ${colorsBlue[7]}`,
-      background: transparentize(0.8, colorsPrimaryAction.textDarkTheme)
+      background: `rgba(${parseToRgb(colorsPrimaryAction.textDarkTheme)}, 0.2)`
```

Comment:
Observation: There's a fair bit of transparentize -> parseToRgb conversion in this PR. It feels like a separate issue to me and a distraction from component style flexibility POC.

Reply:
Probably Can make a separate issue with instructions.